readme.md

```diff
@@ -15,16 +15,24 @@ The clipboard content can also be cleared an shown in browse mode.
 
 Note: The above commands can be changed from NVDA menu, Preferences submenu, Input gestures dialog, Text review category.
 
-## Preferences Menu ##
-*	Clip Contents Designer settings: Allows to set a separator which can be used to find the text segments once the entire added text is pasted.
-It's also possible to choose if the added text will be appended or prepended, if available actions (add, clear clipboard, emulate copy and emulate cut) should be performed inmediately or after confirmation, and if confirmations will be requested always, just if text is contained in the clipboard, or if clipboard is not empty.
-Furthermore, it's possible to change the format and maximum number of characters of the clipboard text which will be shown in browse mode. Please, be aware that increasing this limit may produce issues if the clipboard contains large strings of text. The default limit is 100000 characters.
+## Clip Contents Designer settings ##
+
+This panel is available from NVDA's menu, Preferences submenu, Settings dialog.
+
+It contains the following controls:
+
+* Type the string to be used as a separator between contents added to the clipboard: Allows to set a separator which can be used to find the text segments once the entire added text is pasted.
+* Add text before clip data: It's also possible to choose if the added text will be appended or prepended.
+* Select the actions which require previous confirmation: You can choose, for each action available, if it should be performed inmediately or after confirmation. Available actions are: add text, clear clipboard, emulate copy and emulate cut.
+* Request confirmation before performing the selected actions when: You can select if confirmations will be requested always, just if text is contained in the clipboard, or if clipboard is not empty (for example if you've copied a file, not text).
+* Format to show the clipboard text in browse mode: If you're learning HTML markup language, you may choose Preformatted text in HTML or HTML as shown in a web browser, to have an idea of how your HTML code will be rendered by NVDA in a browser. For example, write some HTML tags like h1, h2, li, pre, etc., select and copy the text to clipboard, and use clipContentsDesigner add-on to show the text in a browseable message. You can change the format to raw text, and then the tags won't be translated in the browser, just will be shown as plain text.
+* Maximum number of characters when showing clipboard text in browse mode: Please, be aware that increasing this limit may produce issues if the clipboard contains large strings of text. The default limit is 100000 characters.
 
 Notes:
 
 *	The above command can be changed from NVDA menu, Preferences submenu, Input gestures dialog, Configuration category.
```

Comment:
The category in documentation was not the same as the one in the code.

```suggestion
*	The above command can be changed from NVDA menu, Preferences submenu, Input gestures dialog, Text review category.
```

Note: Personally, I do not think it is a good idea to have these commands in the Text review category since they can be applied not only in for text review, i.e. in the review cursor context. I would have put them in a category dedicated to this add-on or in the "System status" category, along with "Reports the text on the Windows clipboard".
But this review is focused on the doc, so my suggestion intend to make the doc match the UI.

Reply:
Thanks for your review. About category command to show the add-on settings, currently it is not the same than the other add-on commands. In fact scriptCategory is text review, but this particular command is assigned to config category. I'll create a dedicated category for this add-on, following your suggestion. The code used for this command is:

```
		description=_("Shows the Clip Contents Designer settings."),
		category=SCRCAT_CONFIG
```